### PR TITLE
Add langchainplus-sdk

### PR DESCRIPTION
**Pull request by vowelparrot**
Add recipe for the langchainplus-sdk, generated using grayskull. This contains schemas used in some of the tracers in langchain 

@conda-forge/help-python

Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/langchainplus-sdk/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/langchain-ai/langchainplus-sdk
+  summary: Client library to connect to the LangChainPlus LLM Tracing and Evaluation Platform.
+  license: MIT
+  license_file: PLEASE_ADD_LICENSE_FILE
```
Comment (xhochy) [inline recipes/langchainplus-sdk/meta.yaml:40]:
```suggestion
  license_file: LICENSE
```

**Comment by xhochy** [comment]
You will need to list the langchain entry point in  `build: entry_points: ...`

**Comment by dbast** [comment]
@vowelparrot took the initiative and added the entry_point with test command.

**Comment by dbast** [comment]
@vowelparrot Can we pump the PR to version 0.0.17. That is the version which is required by langchain ==0.0.220, which is a requirement for jupyter-ai in https://github.com/conda-forge/staged-recipes/pull/23554

I would also volunteer as maintainer, if this is of help here.

**Comment by vowelparrot** [comment]
Sure thing. FYI we changed the package name from langchainplus-sdk to langsmith (https://pypi.org/project/langsmith/) which is what is being used in current versions of langchain, and that's already been added to conda, so if/when `jupyter-ai` updates it's langchain pinned version, this would come for free and would be more up-to-date

**Comment by vowelparrot** [comment]
I'll hand off to @hinthornw to clean up

**Comment by dbast** [comment]
ok. right. they already have an issue open to upgrade to newer langchain versions https://github.com/jupyterlab/jupyter-ai/issues/285 ... that would remove the need to add langchainplus-sdk at all.... I'll wait a bit and see how that goes. thanks!

**Comment by hinthornw** [comment]
Even better :) Thank you for your patience!

**Pull request closed without merging**